src/plugins/share/public/components/tabs/link/link_content.tsx

```diff
@@ -112,6 +114,7 @@ export const LinkContent = ({
     copyToClipboard(urlToCopy);
     setUrl(urlToCopy);
     setTextCopied(true);
+    setIsLoading(false);
```

Comment:
not for this pr, but looks like this funciton could benefit from error handling logic. wonder what happens if create short url request fails